docs/USERAPI.md

```diff
@@ -0,0 +1,206 @@
+# User API documentation
+
+This is the documentation for the new user API (and only the new one) that was created in 2020.
+
+## Authorization
+
+### Description
+
+User authorization is been done with OAuth 2.0 and Google as the OAuth 2.0 provider. When calling the authentication URL you will
+be redirected to Google OAuth system (So the user need to be able to see the screen), after Google will authenticate the user he will be redirected
+to https://www.anyway.co.il/authorize/google/callback/google and from there the flow will be redirected to the URL that was given in
+the param `redirect_url` or to the default URL.
+
+### URL struct
+
+> https://www.anyway.co.il/authorize/google
+
+### Method
+
+> GET
+
+### Example
+
+> https://www.anyway.co.il/authorize/google
+
+> http://127.0.0.1:5000/authorize/google?redirect_url=http%3A%2F%2F127.0.0.1%3A5000%2F
```

Comment:
These are local addresses, is this a mistake?

Reply:
No, this is just to indicate that you perform local debugging of this feature(redirect your flow to localhost instead to the Anyway servers).

Reply:
The third example is showing how you can perform local to Anyway servers flow.